hpe3parclient/client.py

```diff
@@ -4658,3 +4658,41 @@ def remoteCopyGroupStatusStoppedCheck(
             return True
         else:
             return False
+
+    def scheduleStatusActiveCheck(
+            self, schedule_name):
+        """
+        Checks whether schedule status is active or not.
+        :param schedule_name - Schedule name
+        :type schedule_name: str
+        :return: True: If schedule status is 'active'
+        :        False: If schedule status in not 'active'
+        """
+        cmd = ['showsched ', schedule_name]
+        try:
+            result = self._run(cmd)
+            for r in result:
+                if 'active' in r:
```

Comment:
Can you paste the response from the CLI if the array is not active?

Reply:
We cant return CLI response, since in ansible we are calling this function, and if this function is returning true then we are not resuming snapshot schedule

Reply:
I want the value of r in both cases to see what it returns, just paste it here. Do not return in code

Reply:
value of r:-
my_ansible_sc,createsv -ro -exp 2h -retain 1h my_ansible_volume.@y@@m@@d@@H@@M@@S@ my_ansible_volume,0,*,*,*,*,3paradm,active,Y,2018-11-27 15:00:00 IST

Reply:
In case of inactive?